sklearn/manifold/isomap.py

```diff
@@ -168,8 +168,8 @@ def _fit_transform(self, X):
         self.embedding_ = self.kernel_pca_.fit_transform(G)
 
     @property
-    @deprecated("Attribute training_data_ was deprecated in version 0.22 and "
-                "will be removed in 0.24.")
+    @deprecated("Attribute `training_data_` was deprecated in version 0.22 and"
```

Comment:
This is a pretty weird thing to warn about. The `training_data_` links: https://scikit-learn.org/dev/modules/generated/sklearn.manifold.Isomap.html#id5